sgx-tools/dcap_linux.go

```diff
@@ -0,0 +1,358 @@
+package main // import "github.com/inclavare-containers/sgx-tools"
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"github.com/go-restruct/restruct"
+	"github.com/golang/protobuf/proto"
+	"github.com/inclavare-containers/rune/libenclave/intelsgx"
+	pb "github.com/inclavare-containers/sgx-tools/proto"
+	"github.com/sirupsen/logrus"
+	"net"
+)
+
+const (
+	aesmdSocket         = "/var/run/aesmd/aesm.socket"
+	attKeyIdLength      = 256
+	nonceLength         = 16
+	qeReportInfoLength  = 960
+	timeOut             = 15000000
+	defaultDcapAttKeyId = "0000000020008c4f5775d796503e96137f77c68a829a0056ac8ded70140b081b094490c57bff0000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
I suggest you use these 3 pb messages GetSupportedAttKeyIDNumRequest, GetSupportedAttKeyIDsRequest and SelectAttKeyIDRequest to handle AK ID instead of this hard coding value.

Reply:
> I suggest you use these 3 pb messages GetSupportedAttKeyIDNumRequest, GetSupportedAttKeyIDsRequest and SelectAttKeyIDRequest to handle AK ID instead of this hard coding value.

ok, LGTM.